src/api/tests/Spread.IntegrationTests/TestFixtures/FollowerTestFixture.cs

```diff
@@ -9,35 +9,35 @@
 
 namespace Spread.IntegrationTests.TestFixtures
 {
-        [TestFixture]
-        public class FollowerTestFixture : IntegrationTestBase
+    [TestFixture]
+    public class FollowerTestFixture : IntegrationTestBase
+    {
+        [SetUp]
+        public Task Setup()
         {
-            [SetUp]
-            public Task Setup()
-            {
-                return Login("test1");
-            }
+            return Login("test1");
+        }
 
-            [Test]
-            [Order(1)]
-            public async Task ICan_List_Followers()
-            {
-                var result = await Api.Get<List<FollowerListDto>>("api/public/follower/followers");
-                Assert.That(result, Is.Not.Null);
-                Assert.That(result.Count, Is.EqualTo(3));
-                Assert.That(result.Any(x=> x.UserName == "test2"), Is.True);
-                Assert.That(result.Any(x => x.UserName == "esengul"), Is.True);
-                Assert.That(result.Any(x => x.UserName == "canperk"), Is.True);
-            }
+        [Test]
```

Comment:
Testleri ById mantığında düzeltmelisiniz

Reply:
Current user için yazılmı aslında